Hf 3 beadás

balog-martin opened the pull request:
Szia 	Csabi, itt a pull-request a 3. Hf beadásához.

Üdv:
Balog Martin

balog-martin committed:
Elsö commit a Hf-3 branchre

balog-martin committed:
Ezek formazasa: body, h1..h6, p, span, div

balog-martin committed:
Tovabbi formazasok: navigacio, div közepen bekeretezve

balog-martin committed:
Form 'Regisztracio' hozzaadva

balog-martin committed:
Kep fejlecbe hozzaadva, plusz nehany style-valtoztatas

balog-martin committed:
'scroll-behavior: smooth' added

balog-martin committed:
headings szine a hatterkephez igazitva

balog-martin committed:
Cikk küldese ürlap hozzaadva

balog-martin committed:
Kep oldal közepen hozzaadva

balog-martin committed:
alt attribute added to img and the emai-input adjusted

balog-martin commented:
Szia @frontendes-csabi, igy remelhetöleg kapsz ertesitest, hogy csinalhatod a review-t illetve az ertekelest :)

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
-                Curabitur viverra nisl dui, a venenatis felis pellentesque id. 
-                Donec non lacus diam.
-            </p>
+        <div class="div_Navigation">
```
Nem praktikus beleírni a classnevekbe a taget, amire alkalmazod. Egyrészt ezt meg tudod oldani szelektorral (`div.navigation`), másrészt, ha kicseréled a taget divről navra monjuk, akkor vagy hazudni fog a class, vagy módosítanod kell a class nevét is a taggel együtt (és mi indokolná a classnév változtatását, amikor struktúrát módosítasz?).
Illetve látok egy aláhúzást, mint elválasztót ebben a classnévben, de a többiben nem, így nem konzisztens a nevezék.

balog-martin replied:
Szia @frontendes-csabi, modositottam (commit 34e3b6a). Igy viszont most van egy `div.Navigation` es `a.Navigation` class. Ez igy okes? Nem problema, hogy mindkettöben a "Navigation" szot hasznalom?

frontendes-csabi replied:
A dolog trükkje az, hogy a classnevek (ha nem utility classokat használsz) jelentést is hordoznak magukkal. Ha egy másik fejlesztő olvassa a HTML forrását és meglátja a `Navigation` classnevet több helyen, akkor egész biztos, hogy azonos jelentést is fog nekik tulajdonítani, ezáltal azonos stílusra is számít. Márpedig, ha jól olvasom a kódot, itt a `div` és az `a` elemeknek más szerepük van, ez extra gondolkodást fog eredményezni a részéről.

De amúgy látom, hogy már módosítottad, szerintem te is érezted ugyanezt :)

balog-martin replied:
Igen, valoban ereztem, h ez igy nem az igazi, de gondoltam rakerdezek. Köszönöm a valszt itt es meg Discordban is!

Igy most `nav.NavigationHorizontal `es `a.NavigationLink` neveket hasznalom. Talan az elözö heten lattam ezt a semat nalad valamelyik oran (?). Ebben tetszik az, hogy definialni tudom, hogy milyen HTML elemre fogom hasznalni. 
Idaig föleg C#-ban dolgoztam, talan abbol jön ez a görcs, hogy hozza vagyok szokva, hogy egy valtozonak ott definialni kell a tipusat.... :)
Emiatt viszont lehet, hogy a JavaScript is fog meg fejfajast okozni :D

frontendes-csabi replied:
Utóbibban sokat fog segíteni a JSDoc vagy a TypeScript, attól függ, melyik passzol a projekthez.

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
-                Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,
-            </p>
+        <div style="
+                width: 100vw;
```
Inline style abszolút ellenjavallt, egy ilyen PR-t munkahelyen nem fogadnék el.

balog-martin replied:
Rendben, modositottam (commit 9e081b9). 

Azert hasznaltam, mert ezek csak egy helyen fordultak elö, es gondoltam megmutatom a hf-ben, hogy az inline style-t is ismerem... :D

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
-            </p>
+        <div style="
+                width: 100vw;
+                height:  600px;
```
```suggestion
                height: 600px;
```

balog-martin replied:
Elfogadva

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
+                        margin: auto;
+                        padding-top: 40px;"> <!-- Ez a style kell, h közepre kerüljön a szöveg...-->              
+                <div>
+                    <h1 class="cimStyle">Frontendes hegyes cím  </h1>   
```
Vegyesen használsz magyar és angol szavakból álló összetételt és a többi classnévvel sincs szinkronban.
A `Style` a classnévben szóismétlés: a classneveket jellemzően arra használjuk, hogy stílust aggassunk az elemekre.

```suggestion
                    <h1 class="cimStyle">Frontendes hegyes cím</h1>   
```

balog-martin replied:
Modositva (commit https://github.com/balog-martin/FrontEndKurzus2023_Hf/commit/9e081b9016871976d4c4f039cb1246675a2b39b1).

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
+                    mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere
                 </p>
+
+                <img    src="Backgrounds/majestic-mountain_small.jpg"
```
Ez egy rosszul tagolt blokk, nem lesz kényelmes a következő fejlesztőnek, aki erre téved.

balog-martin replied:
Javitva (commit [30aefa3](https://github.com/balog-martin/FrontEndKurzus2023_Hf/pull/2/commits/30aefa3b0cc1c6678d991a08232873c0fb4a464c))

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
+                        alt="Hight mountain and sunset"
+                        style=" width: 500px;
+                    display: block;
+                    background-image: url(Backgrounds/majestic-mountain_small.jpg) ;
```
```suggestion
                    background-image: url(Backgrounds/majestic-mountain_small.jpg);
```

balog-martin replied:
Elfogadva, modositva

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
+            <hr>
+            <h1 id="idH1Registration">Regisztráció</h1>
+            <div>
+                <form   onsubmit="return confirm('Biztosan elküldi a jelentkezést?')">
```
```suggestion
                <form onsubmit="return confirm('Biztosan elküldi a jelentkezést?')">
```

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
+                                id="policyAcceptedInput"
+                                required
+                                >
+                        <label  style= "display: inline;
+                                        "
```
```suggestion
                        <label  style="display: inline;"
```

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
+                                type="checkbox" 
+                                name="submitNewsletter" 
+                                id="newsletterInput"  >
+                        <label  style= "display: inline;
+                                        "
```
```suggestion
                        <label  style="display: inline;"
```

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
+            <h1 id="idH1SendArcticle">Cikk küldése</h1>
+            <div>
+                <form name="formF2">
+                    <div >
```
```suggestion
                    <div>
```

frontendes-csabi commented on index.html:
```diff
@@ -1,173 +1,352 @@
+                        <textarea name="messageF2" id="messageTextareaF2"></textarea>
+                    </div>
+                    <div>
+                        <label>*: Kötelezően kitöltendő mező </label>
```
```suggestion
                        <label>*: Kötelezően kitöltendő mező</label>
```

frontendes-csabi commented on style.css:
```diff
@@ -0,0 +1,150 @@
+
+.div_Navigation{
+    font-weight: bold;
+    text-shadow:  rgb(185, 184, 184);
```
```suggestion
    text-shadow: rgb(185, 184, 184);
```

frontendes-csabi commented:
A legtöbb komment főként edukációs jellegű, azaz nem számítottam bele az értékelésbe, egy csomó mindent tapasztalat és gyakorlat fog hozni magával. A PR-t merge-ölheted így is, de azt javasolnám, hogy szabadulj meg az inline stílusoktól, mert azok a következő értékelésnél is szúrni fogják a szememet :)
Örömmel láttam volna, ha kap egy kis szemantikai ízesítést is a html.

balog-martin committed:
A tag-ek neveit kivettem a classnevekböl. Csabi komment: https://github.com/balog-martin/FrontEndKurzus2023_Hf/pull/2#discussion_r1402763051

balog-martin committed:
Inline style modositva external stylera.
Csabi komment: https://github.com/balog-martin/FrontEndKurzus2023_Hf/pull/2#discussion_r1402763051

balog-martin committed:
Image style tagolasa javitva. Inline style-bol modositva external style-ra.

balog-martin committed:
Update index.html

Formazas modositasa elfogadva

Co-authored-by: frontendes-csabi <142182371+frontendes-csabi@users.noreply.github.com>

balog-martin committed:
Update index.html

Formazas modositasa elfogadva 2

Co-authored-by: frontendes-csabi <142182371+frontendes-csabi@users.noreply.github.com>

balog-martin committed:
Update index.html

Formazas modositasa elfogadva

Co-authored-by: frontendes-csabi <142182371+frontendes-csabi@users.noreply.github.com>

balog-martin committed:
Update index.html

Formazas modositasa elfogadva

Co-authored-by: frontendes-csabi <142182371+frontendes-csabi@users.noreply.github.com>

balog-martin committed:
Update style.css

Formazas modositasa elfogadva

Co-authored-by: frontendes-csabi <142182371+frontendes-csabi@users.noreply.github.com>

balog-martin committed:
Update index.html

Formazas modositasa elfogadva

Co-authored-by: frontendes-csabi <142182371+frontendes-csabi@users.noreply.github.com>

balog-martin committed:
Szemantikus HTML: header, footer, nav tagek alkalmazasa

balog-martin committed:
Merge branch 'Hf-3' of https://github.com/balog-martin/FrontEndKurzus2023_Hf into Hf-3

balog-martin committed:
Szemantikus HTML: main tag alkalmazasa
